docs/contributing/metrics.md

```diff
@@ -0,0 +1,37 @@
+# Micrometer metrics in the Splunk distro
+
+This short doc aims to describe the current (legacy) metrics solution in the Splunk distro: how we use Micrometer, which
+parts are placed where (and in which class loader), and how they interact.
+
+Since we're currently not using the OpenTelemetry metrics API & SDK (this will change in the future), we're disabling by
```

Comment:
maybe we're disabling the OTel metrics exporter ... by default in the `SplunkConfiguration` class.